sailfishusermanager.cpp

```diff
@@ -219,8 +219,8 @@ uint SailfishUserManager::addUser(const QString &name)
                 count++;
         }
     }
-    if (count > SAILFISH_USERMANAGER_MAX_USERS) {
-        // Just > above because master user is also member of users group
+    if (count > (SAILFISH_USERMANAGER_MAX_USERS - 1)) {
```

Comment:
Hmm. I can see that you consider master as not user here but if it was considered as an user then this could be `count >= SAILFISH_USERMANAGER_MAX_USERS`.